algorithms/CPlusPlus/README.md

```diff
@@ -191,15 +191,12 @@
 - [Product of two numbers](Recursion\product-of-numbers.cpp)
 - [Product of digits in a number](Recursion\product-of-digits.cpp)
 - [Linear search using recursion](Recursion/linear-search.cpp)
-- [Reverse a number using recursion](Recursion/reverse-a-number.cpp)
 
-## Number System
 
-- [Binary Addition](Number-system/binary_addition.cpp)
-- [Binary to Decimal](Number-system/binary_to_decimal.cpp)
-- [Decimal To Binary](Number-system/decimal_to_binary.cpp)
-- [Decimal To Hexa-Decimal](Number-system/decimal_to_hexa.cpp)
-- [Hexa-Decimal To Decimal](Number-system/hexa_to_decimal.cpp)
-- [Decimal To Octal](Number-system/decimal_to_octal.cpp)
-- [Octal To Decimal](Number-system/octal_to_decimal.cpp)
```

Comment:
Please do not remove the index

Reply:
Index of which file, rod cutting or convex hull??
If i remove the file rod cutting, then how can it have an index sir?
please help me out on this....

Reply:
Ok, sorry, could you merge the our main branch to your this branch? after that add the convex hull in this index file?